### ISSUE TITLE
[Elastic Agent] Documented workflow on how an on-prem user can migrate to Elastic Cloud

### ISSUE THREAD

**Issue by nimarezainia** [elastic/ingest-docs#72]
Need to provide documentation on how a user, who has currently deployed their agents on-premises, can migrate their deployment to be using Elastic Cloud.

background to this request: https://elastic.slack.com/archives/C01QQ449KE1/p1680264788631369

@nimarezainia to provide the content.

**Comment by petericebear** [comment]
Do we have this in the planning?

**Comment by karenzone** [comment]
@nimarezainia @petericebear:  I added this issue to the Platform Ingest docs board with a "Needs PM Definition' label, so it's on the radar. 

@nimarezainia, we can work together on priority and timeframe.

**Comment by karenzone** [comment]
@nimarezainia, how does the work we did in [docs-content#3216](https://github.com/elastic/docs-content/pull/3216) impact this issue?  Topic: https://www.elastic.co/docs/reference/fleet/migrate-elastic-agent#migrate-elastic-agent-migrated-agents

Similar issue (except for Hosted to Serverless): https://github.com/elastic/docs-content/issues/4239

**Comment by nimarezainia** [comment]
@karenzone yes I think those are complete as stated here: https://www.elastic.co/docs/reference/fleet/migrate-elastic-agent#migrate-elastic-agent-migrated-policies  - closing this issue.
